_posts/2015-02-17-aerogear-webpush-alpha1.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: AeroGear WebPush
```

Comment:
A right, I've update it now and it looks right to me. @matzew Can you double check?